downstream/modules/hub/proc-configure-automation-hub-server-gui.adoc

```diff
@@ -8,40 +8,40 @@ Configure Red Hat automation hub as your primary source of content by using the
 * You have obtained the API token for the {HubName} server. See xref:hub-create-api-token[Creating the Red Hat {HubName} API token] for more information.
 [IMPORTANT]
 ====
-	Creating a new token revokes any previous tokens
-	generated for Private Automation Hub. Ensure that you update any Controller or scripts that you created with the previous token.
+Creating a new token revokes any previous tokens generated for {PrivateHubName}. Ensure that you update any {ControllerName} or scripts that you created with the previous token.
 ====
 
 .Procedure
 
-. Navigate to your Automation Controller.
+. Navigate to your {ControllerName}.
 . Create a new credential.
-.. Click btn:[Add] from the **Credentials** screen.
-.. Enter the name for your new credential in the **Name** field.
+.. Navigate to menu:Resources[Credentials].
```

Comment:
Not sure about the UI here, but would it be correct to say instead:

From the navigation panel, select menu:Resources[Credentials].

Reply:
@hherbly  I may rewrite this section because right now we have 2 primary steps with substeps underneath. If I decide to do that, I'll handle it in a separate PR.